yarn.lock

```diff
@@ -4824,7 +4824,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/assets-controllers@npm:^33.0.0":
+"@metamask/assets-controllers@npm:33.0.0":
```

Comment:
Do I need to bump this on mobile as well? https://github.com/MetaMask/metamask-mobile/pull/10562

Reply:
We will bump this on mobile too.  Mobile won't consume this data until the PR you linked, so no hotfix just a regular fix.